e2e-test/cypress/integration/workflowitem_edit_spec.js

```diff
@@ -113,33 +113,3 @@ describe("Workflowitem edit", function() {
     }
   );
 });
-
-describe("Workflowitem create", function() {
```

Comment:
why was this deleted?

Reply:
We already tested it in workflowitem_create_spec. It's duplicate.